modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/persistence/PersistentPageMemory.java

```diff
@@ -275,6 +283,15 @@ public PersistentPageMemory(
         this.writeThrottle = null;
     }
 
+    private void initMetrics() {
+        metricSource.addMetric(new IntGauge(
```

Comment:
Do you think we should document metrics somewhere?

Reply:
In a separate Jira